src/main/java/beans/NewRepeatRequestBean.java

```diff
@@ -0,0 +1,44 @@
+package beans;
+
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonSetter;
+
+/**
+ * Created by willpride on 1/20/16.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class NewRepeatRequestBean {
```

Comment:
Seems identical to `DeleteRepeatRequestBean`, why not just have a `RepeatRequestBean`?